site/docs/v1.0.0/hooks.md

```diff
@@ -77,5 +77,5 @@ velero backup logs nginx-hook-test | grep hookCommand
 
 
 [1]: api-types/backup.md
-[2]: examples/nginx-app/with-pv.yaml
+[2]: https://github.com/heptio/velero/blob/master/examples/nginx-app/with-pv.yaml
```

Comment:
Actually, should this not point to:?

https://github.com/heptio/velero/blob/v1.0.0/examples/nginx-app/with-pv.yaml

Reply:
Yeah -- we can set this one to v1.0, but the problem is there's no way to automatically update these version-specific links as we cut new versions of the docs site - so, for example, when we cut v1.1, we'll copy the `master/` docs into v1.1, and without manually looking through all the docs links, we'll end up with the link pointing to `master` again. Open to ideas here.

Reply:
Can we not use the ``{{ site.latest }}` variable in the URLs?

So `https://github.com/heptio/velero/blob/{{ site.latest }}/examples/nginx-app/with-pv.yaml`.

Reply:
That wouldn't actually solve the copy, problem, though...

Reply:
That does work @nrb, and could be used here.

Reply:
I would not be against having a dup `examples` folder inside `site`.

Reply:
Downside to using {{ site.latest }} is the link no longer works if the page is being viewed directly on GitHub.

We do also package the examples in the release tarball, so maybe we should just remove the link entirely and reference that, since it'll properly match whatever version the user is on.

Reply:
what of ppl who are using something like `brew` and not the tarball directly?

Reply:
Would it make sense to move the examples in under `site/docs`?

Reply:
I'm fine with the `site` directory not working 100% under the GitHub UI, personally.

While we could refer to the examples in the tarball, that also requires the user to stop reading. go download it, extract it, and find the file.

Duplicating the examples under the `site` dir is likely to lead to drift unless we add automation around it. Moving it over entirely might be ok, but viewing the yaml file off of GitHub, as plaintext, isn't a great experience either.

Not really a fan of most of these options at the moment, but I also don't have anything better to offer.

Reply:
If we can add a script to generate a new versioned set of docs from `master/` that handles updating GH links with appropriate versions then I think we'll be OK here. Let me look into that.

Reply:
OK, see https://github.com/heptio/velero/pull/1662 to address https://github.com/heptio/velero/pull/1649#discussion_r302744191. If everyone's happy with that approach then I think this PR can get merged as-is now.